Makefile.am

```diff
@@ -231,10 +239,11 @@ upload-pot:
 download-po:
 	zanata-cli -B pull --min-doc-percent 30 --project-config "$(srcdir)/tools/zanata.xml" -s "$(builddir)/po" -t "$(srcdir)/po"
 
-BOWER = $(srcdir)/tools/bower_components
+BOWER = $(srcdir)/bower_components
 
 update-bower:
-	( cd $(srcdir)/tools && ./bower update )
+	bower update
```

Comment:
we need to change to the source directory here
https://github.com/dperpeet/cockpit/commit/b9c05963681258442a3892b912f4afd680afc912